packages/connect-session/src/browser/index.ts

```diff
@@ -186,28 +95,40 @@ export class ReapitConnectBrowserSession {
           'Content-Type': 'application/x-www-form-urlencoded',
         },
       } as RequestInit)
-      const session = await response.json()
-      if (session.error) {
-        throw new Error(session.error)
-      }
-      if (session) {
-        const { access_token, refresh_token, id_token } = session
-
-        return {
-          accessToken: access_token,
-          // I only get a new refresh token back when grant type is code. I only use grant type code
-          // when I don't have a session, so I can update the refresh token for code and when I have a
-          // session and am refereshing, I can recycle the old refresh token
-          refreshToken: refresh_token ? refresh_token : this.session?.refreshToken,
-          idToken: id_token,
-          loginIdentity: this.deserializeIdToken(id_token),
-        }
+      const session: CoginitoSession | undefined = await response.json()
+
+      if (!session || (session && session.error)) return this.handleError('Error fetching session from Reapit Connect ')
```

Comment:
Suggest to change to (!session ||  !!session?.error)

Reply:
ESLint doesn't like !!